### ISSUE TITLE
[accessibility] no focus state for some interactive elements.

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#363]

**Comment by nisarhassan12** [comment]
@christinfrohne Can you provide me with the design for the focus states for interactive elements ?

**Comment by ChristinFrohne** [comment]
@nisarhassan12 which elements to you mean? 

I think it would be nice to have the selected subject of the navigation bar in blue with the line underneath. So it's easier to see where you currently are. But that's not what you mean, is it?

**Comment by nisarhassan12** [comment]
@ChristinFrohne Sorry! I should have provided you with a brief description of what I meant when I opened the issue. 

Focus state styles mean the styles that get applied when a user focuses over an element usually with a Keyboard (by using the tab key) so if you were to visit https://gitpod.io and start pressing the tab key to navigate you will see that most of the elements don't have focus styles. 

The nav links do have focus styles i.e the blue underline but other elements like links & images that are link and other elements don't have any focus styles.

![image](https://user-images.githubusercontent.com/46004116/72251968-6637ae80-3620-11ea-9819-ef0165169e06.png)

The absence of a focus state makes it difficult for Keyboard users to know on what element they are on the page. 

Does this make sense?

**Comment by ChristinFrohne** [comment]
Okay, thanks for the explanation. Let's use a rectangular box with blue outlines for the focus states. This seems to be the common way. The box should then also be applied to the navigation bar
